Assets/!/Scripts/PlayerAudio.cs

```diff
@@ -0,0 +1,57 @@
+using System.Diagnostics;
+using UnityEngine;
+using utils;
+
+public class PlayerAudio : MonoBehaviour
+{
+    public AudioSource audioSource;
+    public Audio[] audioPlayer;
+    private AudioClip _currentClip;
```

Comment:
We still need to remove the unused attribute

Reply:
It's done, sorry for didn't change completely